cmd/juju/machine/upgradeseries_test.go

```diff
@@ -0,0 +1,99 @@
+package machine_test
+
+import (
+	"bytes"
+	"errors"
+
+	"github.com/juju/cmd"
+	"github.com/juju/cmd/cmdtesting"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/cmd/juju/machine"
+)
+
+type UpgradeSeriesSuite struct {
+}
+
+var _ = gc.Suite(&UpgradeSeriesSuite{})
+
+const machineArg = "1"
+const seriesArg = "xenial"
+
+func (s *UpgradeSeriesSuite) SetUpTest(c *gc.C) {}
+
+func (s *UpgradeSeriesSuite) runUpgradeSeriesCommand(c *gc.C, args ...string) error {
+	err := s.runUpgradeSeriesCommandWithConfirmation(c, "y", args...)
+	return err
+}
+
+func (s *UpgradeSeriesSuite) runUpgradeSeriesCommandWithConfirmation(c *gc.C, confirmation string, args ...string) error {
```

Comment:
Where is the test using --force?

Reply:
To make things simple, I promise to do a drive by test for this one on the next PR. I think this should just be testing that it accepts the flag for now, since we don't actually collect validation info from the machine yet.